htmlbars-version/app/templates/components/focusing-outlet.hbs

```diff
@@ -1,3 +1,3 @@
 {{~#get-outlet-state outletName as |outletState|~}}
   {{outlet outletName}}
-{{/get-outlet-state}}
\ No newline at end of file
+{{/get-outlet-state}}
```

Comment:
Should probably kill the extra newline here, we don't want to add arbitrary whitespace to consuming app's templates (it likely doesn't matter in 99% of cases, but it _could_).